### PR TITLE
update version to 0.6.0-dev

### PR DESCRIPTION
**Pull request by psychocoderHPC**
We missed to update the version after our last release.

### CODE REVIEW

File: include/picongpu/version.hpp [hunk 1]
```diff
@@ -20,6 +20,6 @@
 #pragma once
 
 #define PICONGPU_VERSION_MAJOR 0
-#define PICONGPU_VERSION_MINOR 5
+#define PICONGPU_VERSION_MINOR 6
 #define PICONGPU_VERSION_PATCH 0
 #define PICONGPU_VERSION_LABEL "dev"
```
Comment (ax3l) [inline include/picongpu/version.hpp:25]: did you forget to remove the "dev" to "" in the release, too?

File: share/picongpu/dockerfiles/README.rst [hunk 2]
```diff
@@ -25,7 +25,7 @@ This exposes the ISAAC port to connect via the webclient to.
 .. code:: bash
 
     docker pull ax3l/picongpu
-    docker run --runtime=nvidia -p 2459:2459 -t ax3l/picongpu:0.5.0 lwfa_live
+    docker run --runtime=nvidia -p 2459:2459 -t ax3l/picongpu:0.6.0-dev lwfa_live
```
Comment (ax3l) [inline share/picongpu/dockerfiles/README.rst:28]: I reserved an official `picongpu/...` org on dockerhub a while back as well as a `computationalradiationphysics/` prefix.

Please tell me your dockerhub user names, so I can add you as co-owners and you can then update these lines and push new docker images to it.

Comment (psychocoderHPC) [reply share/picongpu/dockerfiles/README.rst:28]: my dockerhub user name is `psychocoderhpc`

**Comment by psychocoderHPC** [comment]
Independent of what @ax3l wrote the version update can be merged. The docker documentation will be fixed as soon as I have access to dockerhub `computationalradiationphysics`.

File: share/picongpu/dockerfiles/README.rst (hunk 2 above)
Comment (psychocoderHPC) [reply share/picongpu/dockerfiles/README.rst:28]: @ax3l do you reserved it for https://singularity-hub.org/ too?

Comment (psychocoderHPC) [reply share/picongpu/dockerfiles/README.rst:28]: I opened #3544 to keep track that this will be fixed.

**Review by PrometheusPi — DISMISSED** [review]

**Comment by PrometheusPi** [comment]
@psychocoderHPC There are some merge conflict that need to be solved.

File: share/picongpu/dockerfiles/README.rst (hunk 2 above)
Comment (ax3l) [reply share/picongpu/dockerfiles/README.rst:28]: > my dockerhub user name is psychocoderhpc

thanks, added as owner!

> singularity hub

Nope, none there. At some point SH did only allow building on their infrastructure but not pushing pre-build images - and resources to built on their platform were too small to get the whole stack built there. Maybe that changed in between, not sure.

**Comment by psychocoderHPC** [comment]
@PrometheusPi I solved the merge conflicts

**Review by sbastrakov — APPROVED** [review]

**Pull request merged**